tools/cal_scripts/OPTAA/optaa_cal_parser.py

```diff
@@ -9,6 +9,7 @@
 import sys
 import string
 import time
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
what are you trying to do here? code should not be inside the imports and if you're trying to get the path to resolve from the command line for testing, you need an `__init__.py` file in the directory.

Reply:
I have a file called cal_parser_template.py that contains common code shared between the parsers. That line was meant to get a path to the file by referencing the parent directory "cal_scripts" so that way I can use the template code as a package. What would be the best way of having something be a package in the parent directory?